cypress/support/generate.user.js

```diff
@@ -0,0 +1,22 @@
+import { faker } from '@faker-js/faker';
+
+function generateWorker () {
+  const randomNumber = Math.random().toString().slice(4);
+  const firstName = faker.person.firstName() + randomNumber;
+  const lastName = faker.person.lastName();
+  const userEmail = `${firstName}@gmail.com`;
```

Comment:
The email generation logic might cause issues if `firstName` contains special characters or spaces. Consider using a more reliable method to ensure valid email addresses.